app/controllers/v1/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module V1
+  class SessionsController < BaseController
+    skip_before_action :authenticate!, only: :login
+
+    def login
+      @user = User.find_by_email(params[:login][:email])
+      if @user&.authenticate(params[:login][:password])
+        token = JsonWebToken.encode(user_id: @user.id, slug: "hdfc")
+        render_json("User Login Succesfully", auth_token: token)
+      else
+        render_json("Unauthorized User", "User is Not Present", :unauthorized)
+      end
+    end
+
+    def decode_token
+      puts(User.find(1))
+      puts("current", current_user.inspect, current_tenant.inspect)
+    end
```

Comment:
What is the purpose of this method?

Reply:
These method is especially written for the fetching the current user and current tenant from the base controller